financial-dashboard-app/src/app/app.component.html

```diff
@@ -1,5 +1,15 @@
 <main class="main">
-
+  <div class="navigation"></div>
```

Comment:
is het misschien mogelijk dit allemaal in een component te zetten die we dan oproepen in main? Is meer overzichtelijk